src/Protocol/ActivityPub/Processor.php

```diff
@@ -615,8 +616,8 @@ private static function processContent($activity, $item)
 			$content = $activity['content'];
 		} else {
 			// By default assume "text/html"
-			$item['title'] = HTML::toBBCode($activity['name']);
-			$content = HTML::toBBCode($activity['content']);
+			$item['title'] = (!empty($activity['name']) ? HTML::toBBCode($activity['name']) : '');
+			$content = (!empty($activity['content']) ? HTML::toBBCode($activity['content']) : '');
```

Comment:
```suggestion
			$item['title'] = HTML::toBBCode($activity['name'] ?? '');
			$content = HTML::toBBCode($activity['content'] ?? '');
```

Reply:
This suggested change will invoke `HTML::toBBCode()` with an empty `$message` parameter which is completely useless here as to much code runs. I would also suggest to put a check on empty `$message` into `toBBCode()` to avoid executing all that "expensive" code for an empty string.

Reply:
A single check in `HTML::toBBCode` rather than a check for each invocation sounds good to me.

Reply:
I "resolved" both below as they are basically the same as this one.

Reply:
Now that you've added the check for empty string, please simplify these conditions with the null-coalescing operator, including both below.